ibis/backends/tests/test_map.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import duckdb
```

Comment:
You cannot import this here. The tests in `ibis/backends/tests` must be runnable by arbitrary backends without having the dependencies for all backends installed.

For example, let's say I have the polars backend dependencies installed, but not the duckdb backend dependencies and I want to run the backend test suite using polars. Importing `duckdb` here (or the the dependencies for any non-polars backend) will immediately fail with an `ImportError`.

If you need to do something specific to a backend, put that code in `ibis/backend/${specific_backend}/tests`.